.github/workflows/reusable-workflows.yml

```
@@ -13,3 +13,8 @@ jobs:
   pr-branch-target-gitflow:
     name: Check PR for semantic target branch
     uses: mParticle/mparticle-workflows/.github/workflows/pr-branch-target-gitflow.yml@stable
+  security-lint-checks:
+    name: Security Lint Checks
+    uses: mparticle/mparticle-workflows/.github/workflows/security-checks.yml@main
```

I forget the difference, but shouldn't we use `stable` instead of `main`?

```suggestion
    uses: mparticle/mparticle-workflows/.github/workflows/security-checks.yml@stable
```